### ISSUE TITLE
Dependency on `ocaml-migrate-parsetree`

### ISSUE THREAD

**Issue by pitag-ha** [mirage/ocaml-cstruct#315]
Hello :wave:

Is OMP still a test dependency of `ppx_cstruct`? It is according to its `opam` file (both opam-repo and local) and according to the test `dune` file. Looking at the test, it seems to use Ppxlib instead, though, doesn't it?

**Comment by fogti** [comment]
Yeah, appears to work without it. https://github.com/gentoo/guru/pull/144/files#diff-d332b7c85532a6952cb892936b32297595aaf3df7cb02115388420d51c8bc5e1

**Comment by avsm** [comment]
Getting rid of it in #325!
